src/libs/actions/User.js

```diff
@@ -267,6 +266,40 @@ function isBlockedFromConcierge(blockedFromConcierge) {
     return moment().isBefore(moment(blockedFromConcierge.expiresAt), 'day');
 }
 
+/**
+ * Adds a paypal.me address for the user
+ *
+ * @param {String} address
+ */
+function addPaypalMeAddress(address) {
+    const optimisticData = [
+        {
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: ONYXKEYS.NVP_PAYPAL_ME_ADDRESS,
+            value: address,
+        },
+    ];
+    API.write('AddPaypalMeAddress', {
+        value: address,
+    }, {optimisticData});
+}
```

Comment:
I'm actually curious why we are double storing the paypalme address... ?

https://github.com/Expensify/App/blob/13d2766a1988d5119fda6945a16b1eca32594267/src/libs/actions/PersonalDetails.js#L100-L111

I think maybe we would just refer to `personalDetails[currentUserEmail].payPalMeAddress` everywhere. I don't have the full context on why we are storing this in two places.

Reply:
Maybe we can combine them (also maybe makes sense to do this in a follow-up though - not sure)

Reply:
I also agree that this should be a follow up. Considering we have Paypal as part of the payment methods, we shouldn't be calling "UpdatePersonalDetails" from that screen. we can unify it, but we need to decide where it's going to be the better place.